app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -353,6 +353,8 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v2.0.z'
+    registry = Settings.ems.ems_kubernetes.image_inspector_registry || INSPECTOR_REGISTRY_FALLBACK
+    repo = Settings.ems.ems_kubernetes.image_inspector_repository || INSPECTOR_REPOSITORY_FALLBACK
```

Comment:
AFAICT Settings has no API like fetch_path.  Added `.try(:foo)` on all levels.

I dropped the .blank? intentionally.  I did test that
```
:foo:
```
gets parsed as nil (emtpy value in YAML is shorthand for `null`); one has to explicitly write:
```
  :foo: ""
```
to express an empty string.  IMHO that's an explicitly broken config, same as `".#@illegal>"` or `"no-such-registry.com"`...
I generally dislike assigning magical behavior to empty string, unless there is no other way to express it.  Here, if you want to "leave it unset", you can delete the line, or use empty/`null` value.

And remember that we don't really _need_ to support missing options (unless I'm missing something).  It was just simpler than signaling errors.